packages/core/src/utility.ts

```diff
@@ -79,3 +79,14 @@ export const readVarInt = (buffer: Buffer, offset: number = 0): number => {
 export const encodingLength = (n: number) => {
     return n < 0xfd ? 1 : n <= 0xffff ? 3 : n <= 0xffffffff ? 5 : 9;
 };
+
+export const isPubKey = (testVector: Buffer | null): boolean => {
```

Comment:
is there any reason why it should be allowed to accept `null` values?

Reply:
removed